packages/eslint-plugin-pf-codemods/test/testHelpers.js

```diff
@@ -278,7 +283,8 @@ function getInvalidSwapCallbackParamTests(
     componentNameArray,
     propNameArray,
     newParamName,
-    previousParamIndex
+    previousParamIndex,
+    getAddCallbackParamMessage
```

Comment:
I'm a bit confused as to why you didn't add your `createMessageCallback` here?

Reply:
Mainly because I was only utilizing the add callback portion of these helpers rather than the swap callback. Would make sense to allow customizing a message for swap, though, so I can push an update to add that in